c2corg_ui/templates/outing/edit.html

```diff
@@ -23,19 +23,19 @@ <h1 ng-cloak ng-init="id=${outing_id if outing_id else 0}" ng-bind="id ? ('Editi
   % endif 
     name="editForm" novalidate ng-submit="editCtrl.submitForm(editForm.$valid)" class="document-editing">
     
-    ## Init max steps. It won't work with ng-init
+    ## It won't work with ng-init
     <span class="ng-hide">
-      {{editCtrl.max_steps = 5}} 
+      {{editCtrl.max_steps = 4}} 
       {{document = outing}}
+      {{quality_types = [ 'stub', 'medium', 'correct', 'good', 'excellent']}}
```

Comment:
I think it's not possible to keep this as is. The list value cannot be hard-coded.
At least we will have to create an issue and fix this when https://github.com/c2corg/v6_common/issues/16 is done.